pkgs/development/python-modules/pillow/default.nix

```diff
@@ -12,17 +13,25 @@
 
 import ./generic.nix (rec {
   pname = "pillow";
-  version = "9.4.0";
+  version = "9.5.0";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     pname = "Pillow";
     inherit version;
-    hash = "sha256-ocLXeARI65P7zDeJvzkWqlcg2ULjeUX0BWaAMX8c0j4=";
+    hash = "sha256-v1SEedM2cm16Ds6252fhefveN4M65CeUYCYxoHDWMPE=";
   };
 
+  patches = [
+    (fetchpatch {
```

Comment:
I guess this replaces https://github.com/NixOS/nixpkgs/pull/228493/files#diff-9c0332007fcf0d311ac6169461edeb3cc3f44e8bae1e891841a677d19c260981R30-R49 ?

Reply:
It does, as proposed in that PR.